addons/mail/static/src/core/common/composer.js

```diff
@@ -209,17 +210,49 @@ export class Composer extends Component {
     }
 
     get CANCEL_OR_SAVE_EDIT_TEXT() {
-        return markup(
-            _t(
-                "<samp>%(cancel_keybind)s</samp><i> to <a href='#' data-type='%(cancel_type)s'>cancel</a></i>, <samp>%(save_keybind)s</samp><i> to <a href='#' data-type='%(save_type)s'>save</a></i>",
-                {
-                    cancel_keybind: _t("Escape"),
-                    cancel_type: EDIT_CLICK_TYPE.CANCEL,
-                    save_keybind: this.sendKeybind,
-                    save_type: EDIT_CLICK_TYPE.SAVE,
-                }
-            )
-        );
+        if (this.ui.isSmall) {
+            return markup(
+                sprintf(
+                    escape(
+                        _t(
+                            "%(open_button)s%(icon)s%(open_em)sDiscard editing%(close_em)s%(close_button)s"
```

Comment:
this is still not in .pot file for some reason

Reply:
Oops, I forgot `_t()` and added it back, but also forgot to update POT file. Doing it now

Reply:
mail.pot file updated :)